manifests/metacontroller.yaml

```diff
@@ -70,3 +70,11 @@ spec:
         - --logtostderr
         - -v=4
         - --discovery-interval=20s
+        volumeMounts:
+        - mountPath: /etc/ssl/certs
+          name: ca-certs
+          readOnly: true
+      volumes:
+      - hostPath:
+          path: /etc/ssl/certs
```

Comment:
I'm not an expert on how every distro lays out it's certificates but I'm sure they're not always the same.